Sources/Harbor/Mock/HMock.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  HMock.swift
+//  Harbor
+//
+//  Created by Javier Manzo on 06/11/2024.
+//
+
+import Foundation
+
+@HRequestManagerActor
```

Comment:
Was this necessary?

Reply:
If I dont use the actor, I get this error 
<img width="479" alt="Screenshot 2024-11-13 at 3 21 28 PM" src="https://github.com/user-attachments/assets/979b5ca1-1dd9-4f7d-9d93-6fd61cd65fb5">